test/Telegram.Bot.Tests.Integ/appsettings.json

```diff
@@ -7,10 +7,12 @@
 
   "PaymentProviderToken": "",
   "TesterPrivateChatId": "",
+  "Currency": "",
+  "Prices": "",
```

Comment:
These new settings should also be documented in sys integ test wikis.

Reply:
Okay, will document them.